WordPress/Classes/Models/Post.swift

```diff
@@ -133,6 +133,38 @@ class Post: AbstractPost {
         categories = newCategories
     }
 
+    // MARK: - PublicizeConnections
+
+    func publicizeConnectionDisabledForKeyringID(_ keyringID: NSNumber) -> Bool {
+        if let disabled = disabledPublicizeConnections?[keyringID]?["value"],
+            disabled == "1" {
+            return true
+        } else {
+            return false
+        }
+    }
+
+    func enablePublicizeConnectionWithKeyringID(_ keyringID: NSNumber) {
+        if let _ = disabledPublicizeConnections?[keyringID] {
+            if let _ = disabledPublicizeConnections?[keyringID]?["id"] {
+                disabledPublicizeConnections?[keyringID]!["value"] = "0"
```

Comment:
As it's a swift dictionary it's really a struct, so I have to make the assignments in the dictionary itself. However it can be prettier than this, fixed already.